contracts/vaults/Base.sol

```diff
@@ -285,16 +273,20 @@ contract  Base is OwnableUpgradeable, ReentrancyGuardUpgradeable {
             return;
```

Comment:
we can change to `!=` in the previous line, no?

Reply:
probably yes but pls do not use this PR for discussing that :)